resources/etc/nginx/include.d/warp_analytics.conf.tpl

```diff
@@ -18,6 +18,12 @@ if ($http_x_requested_with ~ XMLHttpRequest) {
 	set $warp '</body>';
 }
 
+# Include google analytics scripts if a tracking id is set
+{{ if .Config.Exists "trackingID" }}
+set $analytics '<!-- Global site tag (gtag.js) - Google Analytics --> <script async src="https://www.googletagmanager.com/gtag/js?id={{ .Config.Get "trackingID" }}"></script> <script>   window.dataLayer = window.dataLayer || [];   function gtag(){dataLayer.push(arguments);}   gtag("js", new Date());   gtag("config", "{{ .Config.Get "trackingID" }}"); </script>';
+set $warp '$analytics $warp';
```

Comment:
It looks like you replace the closing body tag regardless of the request type. Please note the tag should only replaced, if the request is a GET or POST request and not an ajax request.